numero/numero.cpp

```diff
@@ -193,9 +193,12 @@ int main(int argc, char** argv) {
 		}
 	} while (i < last_group);
 
-	std::partial_sort(holidays.begin(), holidays.begin() + numero::options::output_lines_number, holidays.end());
+	size_t holidays_size = holidays.size();
```

Comment:
I am trying to follow Mr Stroustrup direction to keep types clean :) std::vector<Holiday>::size() returns std::vector<Holiday>::size_type. Suggest you create a type for this vector where it is defined earlier and use the right type.

Reply:
I used std::vector::size_type directly.